### PR TITLE
Invisible motorbike

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request
New 20 TC item for mime that lets him summon an invisible motorbike and drive on walls with it

# Wiki Documentation
Add the uplink entry when available
# Changelog
:cl:  
rscadd: New 20 TC item for mime that lets him summon an invisible motorbike and drive on walls with it
/:cl:

wip dnm

### CODE REVIEW

**Comment by ynot01** [comment]
If this is WIP please mark it as a draft

**Comment by Moja10** [comment]
Is it possible the mime could bounce Wall riding like in the crazy frog video?

**Comment by Simplehorror** [comment]
> Is it possible the mime could bounce Wall riding like in the crazy frog video?

post the video, i was only basing it off from the Axel F one

File: code/modules/vehicles/invisible_motorbike.dm [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+		AM.density = 0
+		AM.density = 0
+		AM.density = 0
+		AM.density = 0
+		AM.density = 0
+		AM.density = 0
+		AM.density = 0
+		AM.density = 0
+		AM.density = 0
+		AM.density = 0
```
Comment (patpol4) [inline code/modules/vehicles/invisible_motorbike.dm:32]: Are you sure

**Comment by patpol4** [comment]
2/10, doesnt remove the drivers pants

**Comment by bruhlookatthisdood** [comment]
make it run people over like a mule bot

**Comment by Potototatoh** [comment]
i really like this idea

**Comment by Simplehorror** [comment]
this isn't getting merged

**Pull request closed without merging**